### PR TITLE
woo: add stop orders

### PR DESCRIPTION
**Pull request by sc0Vu**
In this PR, I added stop orders for woo. see ccxt/ccxt#17693

### CODE REVIEW

File: ts/src/woo.ts [hunk 1]
```diff
@@ -1070,27 +1288,26 @@ export default class woo extends Exchange {
...
         const price = this.safeString2 (order, 'order_price', 'price');
         const amount = this.safeString2 (order, 'order_quantity', 'quantity'); // This is base amount
         const cost = this.safeString2 (order, 'order_amount', 'amount'); // This is quote amount
         const orderType = this.safeStringLower2 (order, 'order_type', 'type');
-        const status = this.safeValue (order, 'status');
+        const status = this.safeValue2 (order, 'status', 'algoStatus');
         const side = this.safeStringLower (order, 'side');
-        const filled = this.safeValue (order, 'executed');
-        const average = this.safeString (order, 'average_executed_price');
+        const filled = this.safeValue2 (order, 'executed', 'totalExecutedQuantity');
```
Comment (carlosmiei) [inline ts/src/woo.ts:1306]: @sc0Vu let's avoid using `safeValue` when is not strictly required

File: ts/src/woo.ts [hunk 2]
```diff
@@ -825,10 +880,12 @@ export default class woo extends Exchange {
          * @param {float} amount how much of currency you want to trade in units of base currency
          * @param {float|undefined} price the price at which the order is to be fullfilled, in units of the quote currency, ignored in market orders
          * @param {object} params extra parameters specific to the woo api endpoint
+         * @param {bool} params.stop True if is a algo order
+         * @param {string} params.triggerPrice algo order's triggerPrice stopLossPrice = takeProfitPrice = stopPrice = triggerPrice
          * @returns {object} an [order structure]{@link https://docs.ccxt.com/#/?id=order-structure}
          */
         await this.loadMarkets ();
-        const market = this.market (symbol);
+        // const market = this.market (symbol);
```
Comment (carlosmiei) [inline ts/src/woo.ts:888]: @sc0Vu why is this commented?

File: ts/src/woo.ts (hunk 1 above)
Comment (sc0Vu) [reply ts/src/woo.ts:1306]: @carlosmiei what do you mean about this?

File: ts/src/woo.ts (hunk 2 above)
Comment (sc0Vu) [reply ts/src/woo.ts:888]: oh, it's not necessarily to use market params. I'll remove it.

File: ts/src/woo.ts (hunk 1 above)
Comment (carlosmiei) [reply ts/src/woo.ts:1307]: @sc0Vu I mean that we should use `safeString` or `safeNumber` here

**Comment by carlosmiei** [comment]
Was added here: https://github.com/ccxt/ccxt/pull/18379

**Pull request closed without merging**